spec/factories/partnership.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :partnership do
```

Comment:
Do we need some stuff in here? If there's nothing in it then we probably don't need to have this factory

Reply:
I'll remove it as Michal has mentioned below. It'd be helpful to go through some testing with you, especially Factorybot, how it works with traits etc would be good to fully understand and know how best to use.

Reply:
Sure, let me know when you're ready.